python/ray/dashboard/dashboard.py

```diff
@@ -43,6 +43,21 @@ def to_unix_time(dt):
     return (dt - datetime.datetime(1970, 1, 1)).total_seconds()
 
 
+def round_resource_value(quantity):
+    if quantity.is_integer():
+        return int(quantity)
+    else:
+        return round(quantity, 2)
+
+
+def format_resource(resource_name, quantity):
+    if resource_name == "object_store_memory" or resource_name == "memory":
+        # Convert to 100MiB chunks and then to GiB
+        quantity = quantity * (50 * 1024 * 1024) / (1024 * 1024 * 1024)
```

Comment:
Did you mean to use integer division here? The `1024 * 1024` in the numerator will cancel out with one of the `1024 * 1024` occurrences in the denominator in Python 3. Also, why does the comment say `100MiB` but the initial multiplier is `50`?